data_types/src/database_rules.rs

```diff
@@ -181,7 +181,7 @@ pub struct LifecycleRules {
     pub mutable_size_threshold: Option<NonZeroUsize>,
 
     /// Once the total amount of buffered data in memory reaches this size start
-    /// dropping data from memory based on the drop_order
+    /// dropping data from memory based on the [`sort_order`](Self::sort_order)
```

Comment:
I *think* rustdoc is smart enough to figure out the link target in this case. Something like

```suggestion
    /// dropping data from memory based on the [`sort_order`]
```

Reply:
I think that doesn't work with fields, just with types:

## ``[`sort_order`](Self::sort_order)``
![Screenshot_2021-04-13 data_types database_rules LifecycleRules - Rust2](https://user-images.githubusercontent.com/1529400/114512677-a7f74080-9c39-11eb-8e31-6c5e5532dee6.png)

## ``[`sort_order`]`` / `[sort_order]`
![Screenshot_2021-04-13 data_types database_rules LifecycleRules - Rust](https://user-images.githubusercontent.com/1529400/114513018-06242380-9c3a-11eb-95f4-62be8cf810a7.png)